doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -5,13 +5,30 @@ HAP Parameters
 
 In the high-level svm_parameters or mvm_parameters directories, the instrument/detector-dependent configuration index files (e.g., acs_sbc_index.json) provide a map to the configuration files used for the major HAP processing steps. The svm_parameter configuration files are also used for pipeline products.
 
-
-For the astrodrizzle step, there are different files which should be employed depending upon the number of exposures int the visit.
-
+.. _detector_configuration_files:
 
 Detector Configuration Files
 ****************************
 
+For the astrodrizzle step, there are different files which should be employed depending upon the number of exposures int the visit. Examples of these files include names that end in "any_n1" or "any_n2" generally representing the configuration files for the case of one or two exposures. The naming convention, however, can differ for each detector. The following table shows the general naming convention, followed by exceptions to these rules. 
+
+
+.. csv-table:: Pipeline and SVM Alignment Parameter File Conditions
+   :file: align_par_table.csv
+   :header-rows: 2
+
+Exceptions:
+
+    * For MVM processing, if the number of exposures is 1, the corresponding alignment configuration file name will end in "any_1.json". Otherwise it will end in "any_n2" for two or more exposures.
+    * ACS/SBC: if the filters "F115LP" or "F122M" are used, the alignment configuration file will end in "acs_sbc_blue_n2" for 2, 3, 4, or 5 exposures, and "acs_sbc_blue_n6" for 6 or more exposures.
+    * WFC3/IR (grism): if the grism filters "G102" or "G141" are used, the alignment configuration file will end in "wfc3_ir_grism_n2" for two or three exposures, or "wfc3_ir_grism_n4" if there are 4 or more exposures.
```

Comment:
Clarify here that Grism/Prism images are NOT processed for MVM datasets.  Also, they are only partially processed for SVM datasets so the WCS in these images can match the WCS solution in the corresponding _direct_ images (if there are direct images).  See the analyze.py module and comments in hapsequencer.py for more information.

Reply:
I've added another bullet point for the MVM and SVM grism processing.